lib/chromic_pdf/api/pdf_options.ex

```diff
@@ -0,0 +1,117 @@
+defmodule ChromicPDF.PDFOptions do
+  @moduledoc false
+
+  import ChromicPDF.Utils
+  require EEx
+  alias ChromicPDF.ChromeError
+
+  def prepare_export_options(source, opts) do
```

Comment:
If the modules would be aliased instead of imported, the public methods in here might be ok with less verbose names. E.g. `ChromeOptions.build` 🤷 
Also where is `feed_chrome_data_into_output` even used? Didn't find it in the PR. Might because of the import.

Reply:
See that was my whole point about being unsure how to split it up: If it was clearly only about the "ChromeOptions" as an entity, I could also imagine a `ChromeOptions.build`... but the `feed_*` functions aren't concerned so much with the options for chrome anymore, but rather deal with the `output: ` option only. In the end all of these functions are just helpers, completely meaningless without the context of the `API` module. So in the end: Idk, code has been separated, I don't feel like any more renames provide any benefit. Suggestions welcome of course.